.github/workflows/quick-jobs.yml

```diff
@@ -124,4 +124,36 @@ jobs:
       - uses: actions/checkout@v4
       - name: Are buildinfo docs up to date?
         run: make doc/buildinfo-fields-reference.rst
+  release-project:
+    name: Check Release Project
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set PATH
+        run: |
+          echo "$HOME/.cabal/bin" >> $GITHUB_PATH
+      - uses: actions/cache@v3
+        with:
+          path: ~/.cabal/store
```

Comment:
Is this always the correct path? (XDG...)

Reply:
Not sure but maybe for this job it doesn't matter. Being a `cabal build --dry-run` perhaps I don't need the cache?

Reply:
That's s good point.

Reply:
I've removed the cache action from this job.